pkgs/development/tools/build-managers/wafHook/setup-hook.sh

```diff
@@ -11,17 +11,23 @@ wafConfigurePhase() {
     fi
 
     local flagsArray=(
-        @crossFlags@
         "${flagsArray[@]}"
         $wafConfigureFlags "${wafConfigureFlagsArray[@]}"
         ${configureTargets:-configure}
     )
+    if [ -z "${dontAddWafCrossFlags:-}" ]; then
```

Comment:
I didn't see an existing convention for this name, so I made a waf specific one. Would `dontAddCrossFlags` be better?

Reply:
If you want to go for generality, then maybe it's "don't set cross flags". I imagine, some builders could modify existing flags for cross instead of adding some additional ones. Maybe even go for something as vague as `dontSetupCrossBuild`.